### ISSUE TITLE
[BUG] Z Speed

### ISSUE THREAD

**Issue by tslashd** [surftimer/SurfTimer#480]
**Describe the bug**
In short `Z` value does not get reset by start zones. ([video](https://medal.tv/games/csgo/clips/tGmjDOCVj7ICj/d1337vxgBvuG?invite=cr-MSw4NUEsMTM3OTYzMTks))

**Expected behavior**
All zones which have a `Prespeed Limit` set to apply it to `Z` value as well

**Additional context**
Another possible solution to bug from video would be to **not** automatically put player in `Normal` mode by just `OnStartTouch` for start zones and make them type `sm_n` or `sm_r`.

_For now the solution that I have done for this is to put `TeleToStart` zones above platforms which could be exploited like the one from the video (surf_acp_fix)_

**Comment by dPexxIAM** [comment]
It's not easy from the video to tell how he achieves that. I assume he's simply making a saveloc while in noclip? Or using noclip to get above zone and then making a saveloc?

**Comment by tslashd** [comment]
Makes a `saveloc` while in noclip, then just teles to it and he has the speed from the noclip. I'm pretty sure that you get put in `Practice` with that tele but the start zone resets it to `Normal`

**Comment by dPexxIAM** [comment]
Yep - then it's as you said, disabling practice mode bool on client commands rather than StartTouch should do the trick.
